orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -496,7 +503,7 @@ private static boolean isRequired(TypeDescription orcType) {
     }
   }
 
-  private static int getMaxIcebergId(TypeDescription originalOrcSchema) {
+  static int getMaxIcebergId(TypeDescription originalOrcSchema) {
```

Comment:
I don't see this used other than in the conversion logic to assign new IDs. Because that assignment is actually a correctness bug, we don't need this method at all. Also, since it isn't used anywhere else in this PR there is no need to make it package-private instead of private.